spec/Akeneo/Component/SpreadsheetParser/Csv/SpreadsheetSpec.php

```diff
@@ -23,11 +22,11 @@ public function it_returns_the_worksheet_list()
         $this->getWorksheets()->shouldReturn(['sheet']);
     }
 
-    public function it_creates_row_iterators(
-        RowIteratorFactory $rowIteratorFactory,
-        RowIterator $rowIterator1,
-        RowIterator $rowIterator2
-    ) {
+    public function it_creates_row_iterators($rowIteratorFactory, $rowIterator1, $rowIterator2)
+    {
+        $rowIterator1->beADoubleOf('Akeneo\Component\SpreadsheetParser\Csv\RowIterator');
+        $rowIterator2->beADoubleOf('Akeneo\Component\SpreadsheetParser\Csv\RowIterator');
```

Comment:
Thx, i was not aware of that.
I can even use last phpspec 2 as the dependencies seems ok with this project requirements.